### PR TITLE
Add fixed padding option to sentence_embeddings, sentence_indexer

### PR DESCRIPTION
**Pull request by aasseman**
Needed for the attention decoder, as it uses a FC layer that depends on the encoded sequence length.

### CODE REVIEW

File: ptp/components/text/sentence_indexer.py [hunk 1]
```diff
@@ -204,8 +220,21 @@ def tensor_distributions_to_sentences(self, data_dict):
...
+            # Otherwise let torch.nn.utils.rnn.pad_sequence handle it and choose a dynamic padding
+            if self.fixed_padding > 0:
+                pad_list(output_sample, self.fixed_padding)
+
             # Add sentence to batch.
             outputs_list.append(output_sample)
 
         # Create the returned dict.
         data_dict.extend({self.key_outputs: outputs_list})
+
```
Comment (tkornut) [inline ptp/components/text/sentence_indexer.py]: I  guess that function should be removed (as it is already in utils/word_mappings.py

File: ptp/components/utils/word_mappings.py [hunk 2]
```diff
@@ -135,3 +135,21 @@ def save_word_mappings_to_csv_file(logger, folder, filename, word_to_ix, fieldna
             writer.writerow({fieldnames[0]:k, fieldnames[1]: v})
 
     logger.info("Saved mappings of size {} to file '{}'".format(len(word_to_ix), file_path))
+
+def pad_list(l: list, length: int, value = 0):
```
Comment (tkornut) [inline ptp/components/utils/word_mappings.py]: 1) maybe the name should be pad_truncate_list, that will reflect more its behavior?

2) not sure whether word_mappings.py is the right place... but please leave it here for now.

Comment (aasseman) [reply ptp/components/utils/word_mappings.py]: Yeah since the function is used both in `SentenceEmbeddings` and `SentenceIndexer`, I though that it should be somewhere in a "utils" collection of static functions.
